tests/unit/amo/reducers/test_users.js

```diff
@@ -445,4 +450,172 @@ describe(__filename, () => {
+
+      expect(isUnsubscribedFor(state, hash, notification, token)).toEqual(
+        false,
+      );
+    });
+
+    it('does not change the value of another notification', () => {
+      const hash = 'some-hash';
+      const token = 'some-token';
+      const notification1 = 'notification1';
+      const notification2 = 'notification2';
+
+      let state = reducer(
+        undefined,
+        finishUnsubscribeNotification({
+          hash,
+          notification: notification1,
+          token,
+        }),
+      );
+      state = reducer(
+        state,
+        unsubscribeNotification({
+          errorHandlerId: 'error-handler-id',
+          hash,
+          notification: notification2,
+          token,
+        }),
+      );
+
+      expect(isUnsubscribedFor(state, hash, notification1, token)).toEqual(
+        true,
+      );
+      expect(isUnsubscribedFor(state, hash, notification2, token)).toEqual(
+        false,
+      );
+    });
+  });
+
+  describe('finishUnsubscribeNotification', () => {
+    it('sets `isUnsubscribedFor` to `true`', () => {
+      const hash = 'some-hash';
+      const notification = 'new_review';
+      const token = 'some-token';
+
+      const state = reducer(
+        undefined,
+        finishUnsubscribeNotification({
+          hash,
+          notification,
+          token,
+        }),
+      );
+
+      expect(isUnsubscribedFor(state, hash, notification, token)).toEqual(true);
+    });
+
+    it('does not change the value of another notification', () => {
+      const hash = 'some-hash';
+      const token = 'some-token';
+      const notification1 = 'notification1';
+      const notification2 = 'notification2';
+
+      let state = reducer(
+        undefined,
+        unsubscribeNotification({
+          errorHandlerId: 'error-handler-id',
+          hash,
+          notification: notification1,
+          token,
+        }),
+      );
+      state = reducer(
+        state,
+        finishUnsubscribeNotification({
+          hash,
+          notification: notification2,
+          token,
+        }),
+      );
+
+      expect(isUnsubscribedFor(state, hash, notification1, token)).toEqual(
+        false,
+      );
+      expect(isUnsubscribedFor(state, hash, notification2, token)).toEqual(
+        true,
+      );
+    });
+  });
+
+  describe('abortUnsubscribeNotification', () => {
+    it('sets `isUnsubscribedFor` to `true`', () => {
+      const hash = 'some-hash';
+      const notification = 'new_review';
+      const token = 'some-token';
+
+      const state = reducer(
+        undefined,
+        abortUnsubscribeNotification({
+          hash,
+          notification,
+          token,
+        }),
+      );
+
+      expect(isUnsubscribedFor(state, hash, notification, token)).toEqual(null);
+    });
+
+    it('does not change the value of another notification', () => {
+      const hash = 'some-hash';
+      const token = 'some-token';
+      const notification1 = 'notification1';
```

Comment:
As above, to improve readability of the test, you might consider renaming `notification1` to `pendingNotification` and `notification2` to `abortedNotification`.